src/web/components/ClickToView.tsx

```diff
@@ -137,64 +144,59 @@ export const ClickToView = ({
 				className={css`
 					width: 100%;
 					background: ${background.secondary};
-					border: 1px solid ${border.primary};
+					border: 1px solid ${border.secondary};
 					display: flex;
 					flex-direction: column;
 					justify-content: space-between;
-					padding: ${space[3]}px;
-					margin-bottom: 8px;
+					padding: 6px ${space[6]}px ${space[3]}px;
+					margin-bottom: ${space[3]}px;
 				`}
 				data-component={`click-to-view:${sourceDomain}`}
 			>
 				<div
 					className={css`
 						${roleHeadlineSize(role)}
-						margin-bottom: 8px;
+						margin-bottom: ${space[1]}px;
 					`}
 				>
 					{source
 						? `Allow ${source} content?`
 						: 'Allow content provided by a third party?'}
 				</div>
-				<div
+				<p
 					className={css`
 						${textSize}
-						a {
-							${textSize}
-						}
-						p {
-							margin-bottom: 8px;
+						margin-bottom: ${space[5]}px;
+						strong {
+							font-weight: bold;
```

Comment:
I think that this should already be applied by `globalStrongStyles` in `ArticleBody`. If not, that's a flag for a larger issue which might need investigating!

Oh... I'm guessing this is because Storybook doesn't have these globals styles! Myeh, that's annoying. We should look at setting these in config

Reply:
Ive taken this out and added it to the custom styling in the storybook and validated that it gets rendered correctly in the actual article.